src/components/common/SuperMaterialTable.tsx

```diff
@@ -23,13 +23,15 @@ export type UrlQueryParamsType = {
   selection: QueryParamConfig<(string | null | never)[]>;
   sortColumn: QueryParamConfig<number | null | undefined>;
   sortDirection: QueryParamConfig<string | null | undefined>;
+  sortField?: QueryParamConfig<string | null | undefined>;
```

Comment:
Due to the order in which the columns and table are built and rendered, restoring a sort via the column id would result in querying the backend twice when the page initially loads. So I've added the field to the url params to prevent this while still allowing sorts to be applied from the query params.